suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml

```diff
@@ -0,0 +1,213 @@
+#===============================================================================================
```

Comment:
Update testsuite name to `tier1-upgrade-cephadm-without-rgw.yaml`

Reply:
done